superchain/configs/README.md

```diff
@@ -0,0 +1,29 @@
+# configs
+
+The main configuration source, with genesis data, and address of onchain system configuration.
+
+This is a collection of extra configuration data because not all address and system-config information is available through L1 for some chains.
+
+Each chain configuration file in this directory contains configuration data that was generated from the [add-chain](../../add-chain/) code.
+
+- name
+- chain_id 
+- public_rpc
+- sequencer_rpc
+- explorer
+- superchain_level
+- standard_chain_candidate
+- superchain_time
+- system_config_addr
+- batch_inbox_addr
+- genesis:
+  - l1
+    - hash
+    - number
+  - l2:
+    - hash
+    - number
+  - l2_time
+- block_time
+- seq_window_size
```

Comment:
This format has changed, and will likely continue to evolve in the future as we add fields. Maybe we could link to one example like `mainnet/op.toml` so we don't have to update this README whenever we make changes?